extensions/wikia/DesignSystem/styles/DesignSystemGlobalFooter.scss

```diff
@@ -0,0 +1,3 @@
+.mobile-site-link {
+  display: none;
```

Comment:
Indentation should use one tab, not two spaces :)

Reply:
Gotta configure my IDE config then.